src/components/PolicyDetailCard/PolicyDetailCard.tsx

```diff
@@ -0,0 +1,243 @@
+                    setIsOpen={setDropdown}
+                    className="right-[17px]"
+                />
+
+                <PopUpChangePolicyStatus
+                    isOpen={statusPopup}
+                    setIsOpen={setStatusPopup}
+                    policy={policy}
+                    titleSingular={titleSingular}
+                />
+
+                <PopUpAddPolicyToModule
+                    isOpen={modulesPopup}
+                    setIsOpen={setModulesPopup}
+                    policy={policy}
+                    titleSingular={titleSingular}
+                />
+
+                <LeftLineIndicator lastItem={lastItem === true} type={type} />
+
+                <div
+                    className={classNames('left-[2px]', {
+                        'ml-12 relative': type === 'checked out',
+                        'ml-6': type === 'valid' || type === 'archived',
+                    })}>
+                    <Text>{titleSingular}</Text>
+                    <div className="relative z-10 flex-wrap pr-4">
+                        <Heading className="inline mr-3" level="3">
+                            {policy.Titel}
+                        </Heading>
+                        <div className="relative inline-flex -mt-1">
+                            {type === 'archived' ? (
+                                <Badge
+                                    variant="lightGreen"
+                                    className="mr-2"
+                                    upperCase={false}
+                                    text="Vigerend (Buiten werking)"
+                                />
+                            ) : (
+                                <Badge
+                                    variant={
+                                        type === 'valid'
+                                            ? 'lightGreen'
+                                            : 'lightRed'
+                                    }
+                                    className="mr-2"
+                                    upperCase={false}
+                                    text={policy.Status || ''}
+                                />
+                            )}
+                            {policyIsInAModule
+                                ? policy.Ref_Beleidsmodules?.map(module => (
+                                      <Badge
+                                          key={module.UUID}
+                                          variant={'gray'}
+                                          upperCase={false}
+                                          text={module.Titel || ''}
+                                          className="mr-2"
+                                      />
+                                  ))
+                                : null}
+                        </div>
+                    </div>
+                    <div className="mt-2">
+                        {type === 'checked out' ? (
```

Comment:
@Aidenbuis Small thing, but you can use the '&&' operator as well so you don't need to add the null value.

`{type === 'checked out' && ( ... )}`